ibm_catalog.json

```diff
@@ -234,6 +247,12 @@
 							"default_value": "rag-services",
 							"description": "The name of the resource group that is created by this solution. The actual name is prefixed with the value of the input 'prefix'.  All resources created by this solution are deployed in this resource group. ",
 							"required": false
+						},						{
+							"key": "existing_resource_group_name",
```

Comment:
the behavior is not well documented here - what happen when both existing_resource_group_name and resource_group_name are passed?

Reply:
we will address these changes in dev repo (to not increase merge conflicts) with new PR. Issue created https://github.ibm.com/GoldenEye/issues/issues/10462